libs/db/remotedb/grpcdb/server.go

```diff
@@ -162,6 +163,7 @@ func (s *server) handleIterator(it db.Iterator, sendFunc func(*protodb.Iterator)
 
 func (s *server) ReverseIterator(query *protodb.Entity, dis protodb.DB_ReverseIteratorServer) error {
 	it := s.db.ReverseIterator(query.Start, query.End)
+	defer it.Close()
 	return s.handleIterator(it, dis.Send)
```

Comment:
Should we add a comment to `handleIterator` that the caller is responsible for closing the iter?

Reply:
I think it should be obvious since it's passed as an argument, no?

Reply:
Right. I’m just worried that we might forget to close it in other places that will call this (or other) functions.

Reply:
Would be cool to have another program that checks this kind of stuff!